### PR TITLE
Multi-region SQL docs for CREATE/ALTER DATABASE

### PR DESCRIPTION
**Pull request by ericharmeling**
Fixes https://github.com/cockroachdb/docs/issues/9309.
Fixes https://github.com/cockroachdb/docs/issues/10052.
Fixes https://github.com/cockroachdb/docs/issues/9601.
Fixes https://github.com/cockroachdb/docs/issues/10370.
Fixes https://github.com/cockroachdb/docs/issues/9299.
Fixes https://github.com/cockroachdb/docs/issues/10072.
Fixes https://github.com/cockroachdb/docs/issues/9300.
Fixes https://github.com/cockroachdb/docs/issues/9117.
Fixes https://github.com/cockroachdb/docs/issues/10303.
Fixes https://github.com/cockroachdb/docs/issues/10075.
Fixes https://github.com/cockroachdb/docs/issues/10257.
Fixes https://github.com/cockroachdb/docs/issues/10109.
Fixes https://github.com/cockroachdb/docs/issues/9557.

### CODE REVIEW

**Comment by ericharmeling** [comment]
@arulajmani Adding you, Arul, but feel free to loop in @ajstorm and/or @otan.

**Review by ajstorm — CHANGES_REQUESTED** [review]
> Reviewed 10 of 11 files at r1, 1 of 1 files at r2.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/10510)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @arulajmani and @ericharmeling)

___
*[v21.1/configure-zone.md, line 56 at r2](https://reviewable.io/reviews/cockroachdb/docs/10510#-M_0i4fF4_J2rW0WYNq-:-M_0i4fF4_J2rW0WYNq0:b-2ksr6h) ([raw file](https://github.com/cockroachdb/docs/blob/e8c6226109ac219b834981a1947d6d8f710e2b05/v21.1/configure-zone.md#L56)):*
> ```Markdown
> -----------+-------------
> `range_name` | The name of the system [range](architecture/overview.html#glossary) whose [replication zone configurations](configure-replication-zones.html) to change.
> `database_name` | The name of the [database](create-database.html) whose [replication zone configurations](configure-replication-zones.html) to change.<br><span class="version-tag">New in v21.1:</span> If you directly change a database's zone configuration with `ALTER DATABASE ... CONFIGURE ZONE`, CockroachDB will block all [`ALTER DATABASE ... SET PRIMARY REGION`](set-primary-region.html) statements on the database.
> ```

Nit: "to change" -> "are to be changed"? Same comment below.  
___
*[v21.1/create-database.md, line 31 at r2](https://reviewable.io/reviews/cockroachdb/docs/10510#-M_10drSEEQBCIEnq9Ft:-M_10drSEEQBCIEnq9Fu:b-8rgpgb) ([raw file](https://github.com/cockroachdb/docs/blob/e8c6226109ac219b834981a1947d6d8f710e2b05/v21.1/create-database.md#L31)):*
> ```Markdown
> `PRIMARY REGION region_name` | <span class="version-tag">New in v21.1:</span> Create a [multi-region database](multiregion-overview.html) with `region_name` as [the primary region](multiregion-overview.html#database-regions).<br>Allowed values include any region returned by [`SHOW REGIONS FROM CLUSTER`](show-regions.html).
> `REGIONS region_name_list` | <span class="version-tag">New in v21.1:</span> Create a [multi-region database](multiregion-overview.html) with `region_name_list` as [database regions](multiregion-overview.html#database-regions).<br>Allowed values include any region returned by [`SHOW REGIONS FROM CLUSTER`](show-regions.html).<br>To set database regions at database creation, a primary region must be specified in the same `CREATE DATABASE` statement.
> `SURVIVE ZONE FAILURE` (*Default*)<br>`SURVIVE REGION FAILURE` | <span class="version-tag">New in v21.1:</span> Create a [multi-region database](multiregion-overview.html) with regional failure or zone failure [survival goals](multiregion-overview.html#survival-goals).<br>To set the regional failure survival goal, the database must have at least 3 [database regions](multiregion-overview.html#database-regions). To set a regional failure survival goal at database creation, a primary region must be specified in the same `CREATE DATABASE` statement.<br>Surviving zone failures is the default setting for multi-region databases.
> ```

I think we can probably drop "To set a regional failure survival goal at database creation, a primary region must be specified in the same `CREATE DATABASE` statement." as it seems to be transitively implied by the two clauses above.  Alternatively, we can combine this sentence with the previous one, to say "must have at least 3 database regions and a primary region".
___
*[v21.1/create-database.md, line 132 at r2](https://reviewable.io/reviews/cockroachdb/docs/10510#-M_11XPbApN9PLcdLClE:-M_11XPbApN9PLcdLClF:b-1ya32b) ([raw file](https://github.com/cockroachdb/docs/blob/e8c6226109ac219b834981a1947d6d8f710e2b05/v21.1/create-database.md#L132)):*
> ```Markdown
> {% include copy-clipboard.html %}
> ~~~ sql
> > CREATE DATABASE bank PRIMARY REGION "us-east1" REGIONS "us-central1", "us-west1" SURVIVE REGION FAILURE;
> ```

Nit: We _may_ want to show "us-east1" in the REGIONS list too, for completness.
___
*[v21.1/drop-region.md, line 7 at r2](https://reviewable.io/reviews/cockroachdb/docs/10510#-M_14SPS9AXpC1RvGJg3:-M_14SPS9AXpC1RvGJg4:b-8hd1m6) ([raw file](https://github.com/cockroachdb/docs/blob/e8c6226109ac219b834981a1947d6d8f710e2b05/v21.1/drop-region.md#L7)):*
> ```Markdown
> ---
> 
> <span class="version-tag">New in v21.1:</span> The `ALTER DATABASE .. DROP REGION` [statement](sql-statements.html) drops a [region](multiregion-overview.html#database-regions) from a [multi-region database](multiregion-overview.html) with database regions.
> ```

Nit: I think we can drop the last "with database regions".  All multi-region databases have database regions.
___
*[v21.1/drop-region.md, line 35 at r2](https://reviewable.io/reviews/cockroachdb/docs/10510#-M_15Xjb1CyUHoDzFmoj:-M_15Xjb1CyUHoDzFmok:b61sg6q) ([raw file](https://github.com/cockroachdb/docs/blob/e8c6226109ac219b834981a1947d6d8f710e2b05/v21.1/drop-region.md#L35)):*
> ```Markdown
> 
> - To drop any database region, the user must be a member of the [`admin`](authorization.html#roles) or [owner](authorization.html#object-ownership) roles, or have the [`CREATE` privilege](authorization.html#supported-privileges) on the database.
> - To drop the primary (and final) database region, the user must be a member of the [`admin`](authorization.html#roles) or [owner](authorization.html#object-ownership) roles, or have the [`CREATE` privilege](authorization.html#supported-privileges) *and the `ZONECONFIG` privilege* on the database.
> ```

@arulajmani can you confirm these privileges.  I forget what we landed on.
___
*[v21.1/set-primary-region.md, line 16 at r2](https://reviewable.io/reviews/cockroachdb/docs/10510#-M_162lGB3VRO9JzjAw7:-M_162lGB3VRO9JzjAw8:b-3xk6us) ([raw file](https://github.com/cockroachdb/docs/blob/e8c6226109ac219b834981a1947d6d8f710e2b05/v21.1/set-primary-region.md#L16)):*
> ```Markdown
> 
> {{site.data.alerts.callout_warning}}
> If a database's zone configuration has been directly set with an [`ALTER DATABASE ... CONFIGURE ZONE`](configure-zone.html) statement, CockroachDB will block all `ALTER DATABASE ... SET PRIMARY REGION` statements on the database.
> ```

Do we want to discuss the workaround here -> dropping the already applied zone configurations?
___
*[v21.1/set-primary-region.md, line 76 at r2](https://reviewable.io/reviews/cockroachdb/docs/10510#-M_16mOo7GmDd8IqCuPe:-M_16mOo7GmDd8IqCuPf:bmazuty) ([raw file](https://github.com/cockroachdb/docs/blob/e8c6226109ac219b834981a1947d6d8f710e2b05/v21.1/set-primary-region.md#L76)):*
> ```Markdown
> ~~~
> 
> For more information, see [Database regions](multiregion-overview.html#database-regions).
> ```

Can we add an example where we change the primary region?  Also maybe show a failed drop of the primary region when there is more than one region assigned to the database.
___
*[v21.1/sql-statements.md, line 82 at r2](https://reviewable.io/reviews/cockroachdb/docs/10510#-M_173PB18DH43eBE2qe:-M_173PB18DH43eBE2qf:b2udafe) ([raw file](https://github.com/cockroachdb/docs/blob/e8c6226109ac219b834981a1947d6d8f710e2b05/v21.1/sql-statements.md#L82)):*
> ```Markdown
> [`RENAME TABLE`](rename-table.html) | Rename a table or move a table between databases.
> [`SET SCHEMA`](set-schema.html) |  Change the schema of a table.
> [`SET PRIMARY REGION`](set-primary-region.html) | <span class="version-tag">New in v21.1:</span> Change the [primary region](multiregion-overview.html) of a multi-region database.
> ```

Assign a primary region to a database, or change the existing primary region?

**Review by ajstorm — COMMENTED** [review]
Overall, nice work!

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/10510#-:-M_17EGAEbYGzGHp-L37:b734lgp)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @arulajmani and @ericharmeling)

**Review by arulajmani — CHANGES_REQUESTED** [review]
This looks good overall to me as well, mostly some minor comments around the privileges required for ADD/DROP regions. 

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/10510#-:-M_7p--v0CASTfMpf-0V:b-tspi63)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @ajstorm and @ericharmeling)

___
*[v21.1/add-region.md, line 34 at r2](https://reviewable.io/reviews/cockroachdb/docs/10510#-M_7q_Gi3xK1XB1g7n1g:-M_7q_Gi3xK1XB1g7n1h:b-9ktz4d) ([raw file](https://github.com/cockroachdb/docs/blob/e8c6226109ac219b834981a1947d6d8f710e2b05/v21.1/add-region.md#L34)):*
> ```Markdown
> ## Required privileges
> 
> The user must be a member of the [`admin`](authorization.html#roles) or [owner](authorization.html#object-ownership) roles, or have the [`CREATE` privilege](authorization.html#supported-privileges) on the database.
> ```

Same comment about repartitioning Regional By Row tables applies here as well. 
___
*[v21.1/drop-region.md, line 32 at r2](https://reviewable.io/reviews/cockroachdb/docs/10510#-M_7pWkl5Go2lM0Q5POX:-M_7pWkl5Go2lM0Q5POY:b8etwlk) ([raw file](https://github.com/cockroachdb/docs/blob/e8c6226109ac219b834981a1947d6d8f710e2b05/v21.1/drop-region.md#L32)):*
> ```Markdown
> | `region_name`   | The [region](multiregion-overview.html#database-regions) being dropped from this database.  Allowed values include any region present in `SHOW REGIONS FROM DATABASE database_name`. You can only drop the primary region from a multi-region database if it's the last remaining region. |
> 
> ## Required privileges
> ```

Users must also have privileges to repartition all regional by row tables in the database as well. If they're an admin owner, they have them. If on the other hand they were the owner of the database or had the CREATE privilege on the database, then they must also either be the owner of/or have CREATE privilege on all regional by row tables. 

As is, explaining this may be too confusing. Maybe the right answer here is to explain that dropping a region results in changing the partitions on all regional by row tables, which is why you need the privileges to perform that operation? Wdyt?
___
*[v21.1/drop-region.md, line 35 at r2](https://reviewable.io/reviews/cockroachdb/docs/10510#-M_15Xjb1CyUHoDzFmoj:-M_7n0eRAahs5JE1WxYr:b-pvdnci) ([raw file](https://github.com/cockroachdb/docs/blob/e8c6226109ac219b834981a1947d6d8f710e2b05/v21.1/drop-region.md#L35)):*
<details><summary><i>Previously, ajstorm (Adam Storm) wrote…</i></summary><blockquote>

@arulajmani can you confirm these privileges.  I forget what we landed on.
</blockquote></details>

Slight change on this one -- there's nothing special about dropping the primary (and final) region of the database. i.e, just like dropping other regions, the ZONECONFIG privilege doesn't come into the picture. Other than that, this looks good. 

nit: s/the user must be a member of the admin or owner roles/the user must be a member of the admin role or a member of the owner role of the database/g
My intention is to capture the fact that admin is for the entire cluster, but owner is just specific to the database. Feel free to ignore if it feels too wordy. 
___
*[v21.1/set-primary-region.md, line 76 at r2](https://reviewable.io/reviews/cockroachdb/docs/10510#-M_16mOo7GmDd8IqCuPe:-M_7o0C36FG45ZqhzEtP:bf1pxje) ([raw file](https://github.com/cockroachdb/docs/blob/e8c6226109ac219b834981a1947d6d8f710e2b05/v21.1/set-primary-region.md#L76)):*
<details><summary><i>Previously, ajstorm (Adam Storm) wrote…</i></summary><blockquote>

Can we add an example where we change the primary region?  Also maybe show a failed drop of the primary region when there is more than one region assigned to the database.
</blockquote></details>

+1 on showing the failed drop of the primary region if there are other regions on the database. I know we mention it in drop-region.md, but an example to illustrate will go a long way. You could also add it in drop-region.md itself, if you think that flows better.

**Review by ericharmeling — COMMENTED** [review]
Thank you @ajstorm, @arulajmani for the reviews!

I think I got to all your comments.

@arulajmani PTAL at the updates to the "Required Privileges" sections.

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/10510#-:-M_I1mgxCSwPf8bmndT6:bsbpe9f)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @ajstorm and @arulajmani)

___
*[v21.1/add-region.md, line 34 at r2](https://reviewable.io/reviews/cockroachdb/docs/10510#-M_7q_Gi3xK1XB1g7n1g:-M_LUQH6DA850-4UNFT4:bgg9oi4) ([raw file](https://github.com/cockroachdb/docs/blob/e8c6226109ac219b834981a1947d6d8f710e2b05/v21.1/add-region.md#L34)):*
<details><summary><i>Previously, arulajmani (Arul Ajmani) wrote…</i></summary><blockquote>

Same comment about repartitioning Regional By Row tables applies here as well. 
</blockquote></details>

Done.

(Also, I assumed the other comment on `DROP REGION` re: privileges applied to `ADD REGION` as well.)
___
*[v21.1/create-database.md, line 31 at r2](https://reviewable.io/reviews/cockroachdb/docs/10510#-M_10drSEEQBCIEnq9Ft:-M_I11R9-gY-eqaJiQSI:bpe7eqg) ([raw file](https://github.com/cockroachdb/docs/blob/e8c6226109ac219b834981a1947d6d8f710e2b05/v21.1/create-database.md#L31)):*
<details><summary><i>Previously, ajstorm (Adam Storm) wrote…</i></summary><blockquote>

I think we can probably drop "To set a regional failure survival goal at database creation, a primary region must be specified in the same `CREATE DATABASE` statement." as it seems to be transitively implied by the two clauses above.  Alternatively, we can combine this sentence with the previous one, to say "must have at least 3 database regions and a primary region".
</blockquote></details>

Fair enough! Removed that sentence.
___
*[v21.1/create-database.md, line 132 at r2](https://reviewable.io/reviews/cockroachdb/docs/10510#-M_11XPbApN9PLcdLClE:-M_I1GvQ0dIiO1uui-v_:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/e8c6226109ac219b834981a1947d6d8f710e2b05/v21.1/create-database.md#L132)):*
<details><summary><i>Previously, ajstorm (Adam Storm) wrote…</i></summary><blockquote>

Nit: We _may_ want to show "us-east1" in the REGIONS list too, for completness.
</blockquote></details>

Done.
___
*[v21.1/drop-region.md, line 7 at r2](https://reviewable.io/reviews/cockroachdb/docs/10510#-M_14SPS9AXpC1RvGJg3:-M_I1aSt3qtAtHVqwaAn:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/e8c6226109ac219b834981a1947d6d8f710e2b05/v21.1/drop-region.md#L7)):*
<details><summary><i>Previously, ajstorm (Adam Storm) wrote…</i></summary><blockquote>

Nit: I think we can drop the last "with database regions".  All multi-region databases have database regions.
</blockquote></details>

Done.
___
*[v21.1/drop-region.md, line 32 at r2](https://reviewable.io/reviews/cockroachdb/docs/10510#-M_7pWkl5Go2lM0Q5POX:-M_LMxJn1lirvBrK__UI:bkh6k1l) ([raw file](https://github.com/cockroachdb/docs/blob/e8c6226109ac219b834981a1947d6d8f710e2b05/v21.1/drop-region.md#L32)):*
> Maybe the right answer here is to explain that dropping a region results in changing the partitions on all regional by row tables, which is why you need the privileges to perform that operation

IMO, this is closer to how we should explain privileges. But TBH, I think users probably just need to know what privileges they need, and less about why they need them (especially if the "why" has to do with partitioning). I've updated this section to reflect that.
___
*[v21.1/drop-region.md, line 35 at r2](https://reviewable.io/reviews/cockroachdb/docs/10510#-M_15Xjb1CyUHoDzFmoj:-M_I5_6cDygXzZUaTckA:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/e8c6226109ac219b834981a1947d6d8f710e2b05/v21.1/drop-region.md#L35)):*
<details><summary><i>Previously, arulajmani (Arul Ajmani) wrote…</i></summary><blockquote>

Slight change on this one -- there's nothing special about dropping the primary (and final) region of the database. i.e, just like dropping other regions, the ZONECONFIG privilege doesn't come into the picture. Other than that, this looks good. 

nit: s/the user must be a member of the admin or owner roles/the user must be a member of the admin role or a member of the owner role of the database/g
My intention is to capture the fact that admin is for the entire cluster, but owner is just specific to the database. Feel free to ignore if it feels too wordy. 
</blockquote></details>

Done.
___
*[v21.1/set-primary-region.md, line 16 at r2](https://reviewable.io/reviews/cockroachdb/docs/10510#-M_162lGB3VRO9JzjAw7:-M_I2PwK2UuALL_H_QbT:b-5ktdfj) ([raw file](https://github.com/cockroachdb/docs/blob/e8c6226109ac219b834981a1947d6d8f710e2b05/v21.1/set-primary-region.md#L16)):*
<details><summary><i>Previously, ajstorm (Adam Storm) wrote…</i></summary><blockquote>

Do we want to discuss the workaround here -> dropping the already applied zone configurations?
</blockquote></details>

I added a sentence to the end of the note about the workaround.
___
*[v21.1/set-primary-region.md, line 76 at r2](https://reviewable.io/reviews/cockroachdb/docs/10510#-M_16mOo7GmDd8IqCuPe:-M_LNrXN-mu1TBzmo-V-:b69wvd2) ([raw file](https://github.com/cockroachdb/docs/blob/e8c6226109ac219b834981a1947d6d8f710e2b05/v21.1/set-primary-region.md#L76)):*
<details><summary><i>Previously, arulajmani (Arul Ajmani) wrote…</i></summary><blockquote>

+1 on showing the failed drop of the primary region if there are other regions on the database. I know we mention it in drop-region.md, but an example to illustrate will go a long way. You could also add it in drop-region.md itself, if you think that flows better. 
</blockquote></details>

Great idea. Done.

Also added a failed drop example to `drop-database.md`.
___
*[v21.1/sql-statements.md, line 82 at r2](https://reviewable.io/reviews/cockroachdb/docs/10510#-M_173PB18DH43eBE2qe:-M_I1ho82dXYiaRIiPaB:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/e8c6226109ac219b834981a1947d6d8f710e2b05/v21.1/sql-statements.md#L82)):*
<details><summary><i>Previously, ajstorm (Adam Storm) wrote…</i></summary><blockquote>

Assign a primary region to a database, or change the existing primary region?
</blockquote></details>

Done.

**Review by ajstorm — APPROVED** [review]
I've achieved :lgtm: status.  Nice work @ericharmeling!

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/10510#-:-M_QzCqa0rNzkgHHLQiH:b-mh10cy)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @ajstorm, @ericharmeling, and @rmloveland)

File: v21.1/add-region.md [hunk 1]
```diff
@@ -6,12 +6,14 @@ toc: true
 
 <span class="version-tag">New in v21.1:</span> The `ALTER DATABASE .. ADD REGION` [statement](sql-statements.html) adds a [region](multiregion-overview.html#database-regions) to a [multi-region database](multiregion-overview.html).
 
+{% include enterprise-feature.md %}
```
Comment (rmloveland) [inline v21.1/add-region.md:9]: Did you mean "{% include {{page.version.version}}/misc/enterprise-features.md %}" ?

File: v21.1/create-database.md [hunk 2]
```diff
@@ -61,14 +66,19 @@ Parameter | Description
 ~~~
 
 ~~~
-pq: database "bank" already exists
+ERROR: database "bank" already exists
```
Comment (rmloveland) [inline v21.1/create-database.md:69]: dang it's almost like you're really running these commands

File: v21.1/set-primary-region.md [hunk 3]
```diff
@@ -0,0 +1,233 @@
+---
+title: SET PRIMARY REGION
+summary: The SET PRIMARY REGION statement sets the primary region of a multi-region database.
+toc: true
+---
+
+<span class="version-tag">New in v21.1:</span> The `ALTER DATABASE .. SET PRIMARY REGION` [statement](sql-statements.html) sets the primary [region](multiregion-overview.html#database-regions) of a [multi-region database](multiregion-overview.html).
+
+{% include enterprise-feature.md %}
```
Comment (rmloveland) [inline v21.1/set-primary-region.md:9]: Same as above re: did you mean "{% include {{page.version.version}}/misc/enterprise-features.md %}"  ?

**Review by rmloveland — APPROVED** [review]
LGTM!  Most comments are nice-to-haves, not essential. 

I was curious about that one include link's syntax but the build is green so maybe it's working as expected (?)

**Review by ericharmeling — COMMENTED** [review]
TFTR @rmloveland !  I think I got to all your comments.

The include is correct; there are just two files with a very similar name, at different levels in the `_includes` directory.

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/10510#-:-M_Vg8yp1rhCbSEa6h5s:b-34f75v)* status: :shipit: complete! 0 of 0 LGTMs obtained (and 1 stale) (waiting on @ajstorm, @arulajmani, @ericharmeling, and @rmloveland)

___
*[v21.1/add-region.md, line 9 at r4](https://reviewable.io/reviews/cockroachdb/docs/10510#-MZxIxeh3cnF4v1GjzCr-r4-9:-M_VdsDEDpF8S1x46qpK:b5ojkqm) ([raw file](https://github.com/cockroachdb/docs/blob/41a2c8c0eca1a73ddfc1ad158fe20246e8efe9f3/v21.1/add-region.md#L9)):*
<details><summary><i>Previously, rmloveland (Rich Loveland) wrote…</i></summary><blockquote>

Did you mean "{% include {{page.version.version}}/misc/enterprise-features.md %}" ?
</blockquote></details>

I did not. There is an include file that has a very similar name, at the top level of the `_includes` directory (`_includes/enterprise-feature.md`).... It's actually used by more pages than `{{page.version.version}}/misc/enterprise-features.md`, but IDK why in the world we have two separate ones.
___
*[v21.1/create-database.md, line 69 at r4](https://reviewable.io/reviews/cockroachdb/docs/10510#-MZxIxeh3cnF4v1GjzCu-r4-69:-M_VfV2q777YU4xtGIE4:b-dctq58) ([raw file](https://github.com/cockroachdb/docs/blob/41a2c8c0eca1a73ddfc1ad158fe20246e8efe9f3/v21.1/create-database.md#L69)):*
<details><summary><i>Previously, rmloveland (Rich Loveland) wrote…</i></summary><blockquote>

^ this is a joke
</blockquote></details>

:laughing:
___
*[v21.1/drop-region.md, line 9 at r4](https://reviewable.io/reviews/cockroachdb/docs/10510#-MZxIxeh3cnF4v1GjzCv-r4-9:-M_VfbvJCDwt0RckNgO7:b-apayz0) ([raw file](https://github.com/cockroachdb/docs/blob/41a2c8c0eca1a73ddfc1ad158fe20246e8efe9f3/v21.1/drop-region.md#L9)):*
<details><summary><i>Previously, rmloveland (Rich Loveland) wrote…</i></summary><blockquote>

Same q as above re: did you mean "{% include {{page.version.version}}/misc/enterprise-features.md %}"  ?
</blockquote></details>

See my comment above.
___
*[v21.1/set-primary-region.md, line 9 at r4](https://reviewable.io/reviews/cockroachdb/docs/10510#-MZxIxeh3cnF4v1GjzCw-r4-9:-M_VfexxBTFZSrQ6VfOh:b-apayz0) ([raw file](https://github.com/cockroachdb/docs/blob/41a2c8c0eca1a73ddfc1ad158fe20246e8efe9f3/v21.1/set-primary-region.md#L9)):*
<details><summary><i>Previously, rmloveland (Rich Loveland) wrote…</i></summary><blockquote>

Same as above re: did you mean "{% include {{page.version.version}}/misc/enterprise-features.md %}"  ?
</blockquote></details>

See my comment above.
___
*[v21.1/set-primary-region.md, line 16 at r4](https://reviewable.io/reviews/cockroachdb/docs/10510#-MZxIxeh3cnF4v1GjzCw-r4-16:-M_VffLMCl0kcwagXE_J:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/41a2c8c0eca1a73ddfc1ad158fe20246e8efe9f3/v21.1/set-primary-region.md#L16)):*
<details><summary><i>Previously, rmloveland (Rich Loveland) wrote…</i></summary><blockquote>

first mention of zone config docs could be a link to 'em
</blockquote></details>

Done.
___
*[v21.1/set-primary-region.md, line 39 at r4](https://reviewable.io/reviews/cockroachdb/docs/10510#-M_Qhe6tECzaINgvbsap:-M_VgwtS1tUZzz24Prx7:byof0eg) ([raw file](https://github.com/cockroachdb/docs/blob/41a2c8c0eca1a73ddfc1ad158fe20246e8efe9f3/v21.1/set-primary-region.md#L39)):*
<details><summary><i>Previously, arulajmani (Arul Ajmani) wrote…</i></summary><blockquote>

It may be worth calling out that you need `CREATE` or be a member of the `OWNER` role for all tables in the database _only_ when adding the primary region for the first time. 

Thereafter, switching primary regions only requires permissions (CREATE or member of OWNER role) on the database. 
</blockquote></details>

Good catch!

Done.
___
*[v21.1/set-primary-region.md, line 67 at r4](https://reviewable.io/reviews/cockroachdb/docs/10510#-MZxIxeh3cnF4v1GjzCw-r4-67:-M_Vfg7EBudGQyhjpb3n:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/41a2c8c0eca1a73ddfc1ad158fe20246e8efe9f3/v21.1/set-primary-region.md#L67)):*
<details><summary><i>Previously, rmloveland (Rich Loveland) wrote…</i></summary><blockquote>

link to ADD REGION doc?
</blockquote></details>

Done.
___
*[v21.1/sql-statements.md, line 37 at r4](https://reviewable.io/reviews/cockroachdb/docs/10510#-MZxIxeh3cnF4v1GjzCx-r4-37:-M_VgtXNDRuPt4_1J8GN:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/41a2c8c0eca1a73ddfc1ad158fe20246e8efe9f3/v21.1/sql-statements.md#L37)):*
<details><summary><i>Previously, rmloveland (Rich Loveland) wrote…</i></summary><blockquote>

This and the following two sections that talk about 'add a region' could use the slightly more specific `multiregion-overview.html#database-regions` anchor link
</blockquote></details>

Done.
